### PR TITLE
Added default table for password reset tokens

### PR DESCRIPTION
**Pull request by adrean-hipe**
OVERVIEW

- added the default table for password reset tokens
- added timestamps for the users table

NOTES

- the new table is required by the 'reset password' feature of the login by default
- the new column on users table is required for resetting the password - updated_at

### CODE REVIEW

File: laravel-app/database/migrations/2023_09_21_052219_create_password_reset_tokens_table.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::dropIfExists('password_reset_tokens');
+    }
+};
```
Comment (hipe01mark) [inline laravel-app/database/migrations/2023_09_21_052219_create_password_reset_tokens_table.php:28]: Add new line

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

File: laravel-app/database/migrations/2023_09_19_021715_create_users_table.php
Comment (nadj1233) [inline laravel-app/database/migrations/2023_09_19_021715_create_users_table.php:1]: maybe create new migration file for this

File: laravel-app/database/migrations/2023_09_19_021715_create_users_table.php [hunk 2]
```diff
@@ -18,6 +18,7 @@ public function up(): void
             $table->boolean('is_verified')->default(false);
             $table->string('password');
             $table->rememberToken();
+            $table->timestamps();
```
Comment (bpocrafael) [inline laravel-app/database/migrations/2023_09_19_021715_create_users_table.php:21]: why you updated this??? please dont update the previous migration files..

**Review by bpocrafael — CHANGES_REQUESTED** [review]

File: laravel-app/database/migrations/2023_09_19_021715_create_users_table.php (hunk 2 above)
Comment (adrean-hipe) [reply laravel-app/database/migrations/2023_09_19_021715_create_users_table.php]: removed changes on users migration file, added new file to add timestamps column on existing users table

**Review by bpocrafael — APPROVED** [review]
LGTM 🚢

**Review by nadj1233 — APPROVED** [review]
LGTM ✨

**Pull request merged**